WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -133,7 +133,7 @@
             <!--Quick Start-->
             <LinearLayout
                 android:id="@+id/row_quick_start"
-                android:visibility="gone"
+                android:visibility="visible"
```

Comment:
it should be `visible` by default, right? so you can remove this line

Reply:
No, visibility is conditioned (not visible for the majority of users). I made it explicitly visible for testing purposes.